.github/workflows/cd.yaml

```diff
@@ -155,8 +155,17 @@ jobs:
     if: github.event_name == 'release'
     strategy:
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.6, 3.7, 3.8, 3.9]
         os: [ubuntu, macos, windows]
+        exclude:
```

Comment:
the reason is mostly following the previous suggestion from here https://github.com/neuro-inc/neuro-extras/issues/180
probably, due to resources savings, but I'm not sure.
@asvetlov , mb you could give us a hint?

Reply:
@anayden , so the main reason is not to overload the platform (specifically since we are running e2e against staging cluster and control plane) - from Andrew.

In this excluding combinations, there is one constantly failing thing: Colorama doesn't want to install on python 3.7 on windows (maybe, skipping this combination is even better for us 😆 )

In the future, I see the following plan: (1) we migrate our e2e tests to the dev cluster, (2) we scale up dev cluster to fit all those tests, and (3) we turn back those tests.
I've left a corresponding comment to ci/cd description in 0691baf96d8803e4dd454a8542f0b09a031dcd64

Reply:
Thanks for digging to the bottom of that!